### DISCUSSION TITLE
Flux namespace deleted, but kustomization and gitRepository resources still showing up.

### DISCUSSION THREAD

**Discussion by AnasKhan0607** [fluxcd/flux2#3516] [Q&A] [answered]
We are trying to delete Flux. We started of by deleting the namespace, we did so by patching the finalizer out by using `kubectl get ns flux-system -o json | jq '.spec.finalizers = []' | kubectl replace --raw "/api/v1/namespaces/flux-system/finalize" -f -` .  We expected that after deleting the namespace, we wouldn't see any flux resources left over. This was not the case as we still see kustomization and gitRepository resources left over.  

We are aware that there is a finalizer on these resources and we have tried to patch it out using the same command but we are unable to do so due to the fact that the namespace has already been deleted. The error: `error: [kustomizations.kustomize.toolkit.fluxcd.io](http://kustomizations.kustomize.toolkit.fluxcd.io/) “flux-system” could not be found on the server
[kustomization.kustomize.toolkit.fluxcd.io/flux-system-common](http://kustomization.kustomize.toolkit.fluxcd.io/flux-system-common) skipped
The edits you made on deleted resources have been saved to “/var/folders/p1/y7lt6c0n1mn8_66zrsp20d4rr7ryg8/T/kubectl-edit-zzsw8.yaml”`

Has anyone encountered this situation before?

**Comment by stefanprodan** [comment, marked as answer]
You can't delete the namespace to uninstall Flux, please run `flux uninstall` that does the right thing by removing finalizers and CRDs.

**Reply by SharifiFaranak** [reply]
that makes sense, thank you. 

Since we have already deleted the namespace, `flux uninstall` gives the same error about the namespace not being found. Would you know if re-creating the namespace could potentially fix the problem?

**Reply by SharifiFaranak** [reply]
re-creating the namespace then running `flux uninstall` worked 👍
